src/api/web/index.js

```diff
@@ -43,6 +43,25 @@ module.exports = function(config, auth, storage) {
   const indexTemplate = path.join(themePath, 'index.html');
   const template = fs.readFileSync(indexTemplate).toString();
 
+  // Logo
+  let logoURI = _.get(config, 'web.logo') ? config.web.logo : '';
+  if (logoURI && !/^(https?:)?\/\//.test(logoURI)) {
```

Comment:
This `!/^(https?:)?\/\//.test(logoURI)` should be on utilities `src/lib/utils.js`. Export a function so we can reuse it later.

Reply:
Please include couple of unit test. `test/unit/api/utils.spec.js`

Reply:
Done.